src/index.html

```diff
@@ -7,6 +7,419 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo"
+        >
+      </a>
+      <nav>
+        <ul class="nav">
+          <li class="nav__item is-active">
+            <a class="is-active" href="#">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a>
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a>
+              Smartphones
+            </a>
+          </li>
+
+          <li data-qa="nav-hover" class="nav__item">
+            <a>
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a>
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a>
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a>
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a>
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div data-qa="card" class="card">
+        <img 
+          class="card__img" 
+          src="images/imac.jpeg" 
+          alt="imac"
+        >
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+  
+        <p class="card__pcode">
+          Product code: 195434
+        </p>
+  
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__reviews-count">
+            Reviews: 5
+          </div>
```

Comment:
use it in all cases
![image](https://user-images.githubusercontent.com/91411692/222699390-e5084274-7bc7-48db-90a7-24f24e2f4c9e.png)

```suggestion
          </div>
          
          <div class="card__reviews-count">
            Reviews: 5
          </div>
```